packages/flutter/test/cupertino/button_test.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Layout minimum size', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      new Center(child: new CupertinoButton(child: new Text(' '), onPressed: null))
+    );
+    RenderBox buttonBox = tester.renderObject(find.byType(CupertinoButton));
+    expect(
+      buttonBox.size,
+      greaterThanOrEqualTo(const Size.square(48.0)),
```

Comment:
If you're worried about text metrics, I'd just put a SizedBox as the child instead of text.  That will give you exact control over what you're testing.